cfpb/cfgov-refresh
# Enable output of Resource snippet thumbnails in Snippet Lists

Per https://[GHE]/CFPB/hubcap/issues/54, we want to enable the dormant `thumbnail` field of Resource snippets for output in Snippet List modules.

## Additions

- A boolean option to show thumbnails for all snippets in a Snippet List
- Template code to output snippet thumbnails in a Snippet List if the above option is selected
- Some necessary styling to enforce the thumbnail width of 150px

## Changes

- Fixes the part where I was stupidly overriding Wagtail's native `image` template tag
- Fixes a few whitespace mistakes (both in rendered styles and in code)

## Testing

1. Pull branch
2. Migrate
3. Run server
4. Add a thumbnail to a Resource snippet
5. Set up a Snippet List that will pull in that Resource
6. Check out the rendered thumbnail in the Snippet List

## Checklist
